### ISSUE TITLE
automatically show the "links" (like {{hint:Sources}})

### ISSUE THREAD

**Issue by huandney** [thiswillbeyourgithub/Oliver_Cloze#5]
Hello, how long ...
I don't know if it is a characteristic, but when we click on the top, with the mouse, the {{hint: Sources}} fields are not opened automatically.
Obs: 
1. When I use shortcuts or buttons (with the exception of <button> clear) everything works perfectly.
2. In ankidroid, it only works with the buttons, with the exception the clear button again.

![test](https://user-images.githubusercontent.com/19948348/112845062-6af45100-9072-11eb-9cc1-d4ba8d725543.gif)

**Comment by thiswillbeyourgithub** [comment]
Hi!

Indeed the code currently won't open the links if I click on a rectangle. I didn't think about it so it's not really by design but I don't really ever click on the cloze so I never minded.

If it's a huge issue for you I could add this on the todolist but I don't feel it's worth spending time on this.

**Comment by huandney** [comment]
If it's not a lot of work, I find it very interesting, mainly because of the ankidroid, which most of the time I use only the rectangle.
Thanks.

**Comment by thiswillbeyourgithub** [comment]
Okay, I'll add it to the todo. I think I will do it in under a week but I have a lot of things going on.

**Comment by thiswillbeyourgithub** [comment]
Well surprisingly it was actually super simple to implement, just a copy and paste. 

I just pushed it with also additionnal improvements (check out the list of commit to know more). Please test this and tell me if it works.

Also do you happen to know if it works in anki version after 2.1.35 ? I still haven't updated because I fear I won't have the time to deal with all the addon breakage...

**Comment by huandney** [comment]
Both on ankidroid and windows, for some reason it didn't work. :(

![test2](https://user-images.githubusercontent.com/19948348/112981975-e82ecd00-9129-11eb-9ef5-85aaceafaca5.gif)

> Also do you happen to know if it works in anki version after 2.1.35 ? I still haven't updated because I fear I won't have the time to deal with all the addon breakage...

I don't know, I use that version too, for the same reason.

**Comment by thiswillbeyourgithub** [comment]
Sorry I wasn't clear. I pushed it to the separate front.js and back.js files, not to the template file as it something I can't really automate.

So just paste the content of those files to your template and report back to me. But don't forget to backup yours if you modified them (I changed a few things like button name, shortcuts etc...) :)

**Comment by huandney** [comment]
Working perfectly!
Would you like to see my model? I added some nice things.

**Comment by thiswillbeyourgithub** [comment]
Sure! Please open a new issue and explain your changes in plain english then add the code. That would save me some time :)

**Comment by thiswillbeyourgithub** [comment]
@huandney 

Also to add to what we were saying, I figured why I actually never manually click on the rectangles. I assigned the rectangles a pretty small width so that you can't really see if it's 4 cloze next to one another or just one.

So when clicking on the rectangle you can mistakenly click on the second rectangle from the left instead of the first one for example.

Whereas using the cloze button and shortcuts avoids this issue entirely and opens the cloze one by one in the right order.

Hope it more clear.

**Comment by huandney** [comment]
> @huandney
> 
> Also to add to what we were saying, I figured why I actually never manually click on the rectangles. I assigned the rectangles a pretty small width so that you can't really see if it's 4 cloze next to one another or just one.
> 
> So when clicking on the rectangle you can mistakenly click on the second rectangle from the left instead of the first one for example.
> 
> Whereas using the cloze button and shortcuts avoids this issue entirely and opens the cloze one by one in the right order.
> 
> Hope it more clear.

I also usually only use shortcuts, but on the ankidroid I prefer to touch the rectangle.
Thanks for incrementing.
